cmd/directories.go

```diff
@@ -71,19 +61,39 @@ func ParseFromCmdArgs(flags *flag.FlagSet, options *DirectoryOptions) error {
 	return nil
 }
 
-// Check checks that configured dirs exist (which should contain info) and runtime dirs are created (if not exist)
-func (options *DirectoryOptions) Check() error {
-	err := ensureDirExists(options.Config)
+// RegisterDirectoryFlags function register directory options to flag set
+func RegisterDirectoryFlags(flags *[]cli.Flag, options *node.NodeOptions) error {
```

Comment:
`*` for `flags` can be omitted here, slice already a reference type.

Reply:
Tried, but did not succeed pass wo reference sign &

Reply:
just removing all `&` and `*` solves the problem.
like following: 
![image](https://user-images.githubusercontent.com/8612618/44535214-5497f300-a71b-11e8-9184-bb3659500b98.png)

Reply:
Yes I tried, but in that cases, values is appended to copies :/

Reply:
OK, got it. Sorry for confusing.